sbnanaobj/StandardRecord/SRHelper.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SBNANAOBJ_STANDARDRECORD_SRHELPER_H
```

Comment:
Not sure I like `SRHelper` as a name. Maybe `SRConstants`?

Reply:
Done - checked only in that it builds.